### PR TITLE
Update master

### PR DESCRIPTION
**Pull request by mtrabelsi**
its a copy of window branch except for specific project config is ignored.

### CODE REVIEW

File: src/CreateNewPage/index.js [hunk 1]
```diff
@@ -54,8 +59,36 @@ class CreateItemPage extends Component {
       !this.type.value ||
       this.departure.value === this.destination.value
     );
+=======
+      idError: !this.state.id,
+    });
+
+    return !this.state.id;
+>>>>>>> 26faeee... Got rid of Ref and make the component Controlled
```
Comment (mtrabelsi) [inline src/CreateNewPage/index.js]: silent conflict?

File: src/CreateNewPage/index.js [hunk 2]
```diff
@@ -140,57 +166,29 @@ class CreateItemPage extends Component {
...
+                onChange={this.handleTextChange}
+                id="itemId"
+                label={`${unit} ID`}
+                required
+                type="text"
+                error={idError}
+                errorText="This field is required."
+              />
+
+              <Button onClick={this.startScanner} raised primary swapTheming>Start</Button>
```
Comment (sobolev-alexey) [inline src/CreateNewPage/index.js]: please add flag to Firebase config, as discussed earlier

File: src/CreateNewPage/index.js [hunk 3]
```diff
@@ -140,57 +166,29 @@ class CreateItemPage extends Component {
...
-              required
-              label="Departure Port"
-              className="md-cell"
-              menuItems={PORTS}
-              position={SelectField.Positions.BELOW}
-              error={departureError}
-              errorText="This field is required."
-              dropdownIcon={<FontIcon>expand_more</FontIcon>}
-            />
-            <SelectField
```
Comment (sobolev-alexey) [inline src/CreateNewPage/index.js]: why these fields were removed?

Comment (mtrabelsi) [reply src/CreateNewPage/index.js]: we agreed to only ignore the specific configuration of the master branch, that is why this change coming from window

**Comment by mtrabelsi** [comment]
@sobolev-alexey : good to go ?

**Comment by sobolev-alexey** [comment]
@mtrabelsi not yet, I don't see the flag `qrReader: false` added to the DB schema file `/config/item-tracking-export.json`

File: .gitignore [hunk 4]
```diff
@@ -14,3 +14,4 @@
 /firebase_functions/appengine
 
 *.log
+yarn.lock
```
Comment (sobolev-alexey) [inline .gitignore]: please revert this change, this file should be update every time a package is changed or added
